web/context.go

```diff
@@ -39,18 +39,21 @@ type RequestEvent struct {
 	Contacts       []string `json:",omitempty"`
 	UserAgent      string   `json:"ua,omitempty"`
 	// Origin is sent by the browser from XHR-based clients.
-	Origin  string                 `json:",omitempty"`
-	Payload string                 `json:",omitempty"`
-	Extra   map[string]interface{} `json:",omitempty"`
+	Origin string                 `json:",omitempty"`
+	Extra  map[string]interface{} `json:",omitempty"`
```

Comment:
I believe we're no longer using `Extra`.

Reply:
We are:

```
wfe2/wfe.go
817:    logEvent.Extra["CertificateSerial"] = serial
819:            logEvent.Extra["RevocationReason"] = *revokeRequest.Reason
1613:   logEvent.Extra["RequestedSerial"] = serial
2244:   logEvent.Extra["KeyType"] = web.KeyTypeToString(csr.PublicKey)
2319:   logEvent.Extra["RequestedSerial"] = serial
```